piet/Cargo.toml

```diff
@@ -12,12 +12,12 @@ categories = ["rendering::graphics-api"]
 include = ["src/**/*", "Cargo.toml", "snapshots/resources/*"]
 
 [dependencies]
-image = { version = "0.24.0", optional = true, default-features = false }
-kurbo = "0.8.2"
-pico-args = { version = "0.4.0", optional = true }
-png = { version = "0.17", optional = true }
-os_info = { version = "3.0.0", optional = true, default-features = false }
-unic-bidi = "0.9"
+image = { version = "0.24.2", optional = true, default-features = false }
+kurbo = "0.8.3"
```

Comment:
Do we want to pull off the bandaid and jump to 0.9 now that's been released? I don't *think* there are any client changes that will be impacted, and I confirm the example compiles and runs without modification.

Reply:
May as well give it a try. I guess it has the downside of requiring a recent `rustc` but I don't think we've ever cared about supporting old compilers, right?

Reply:
I think we're happy to be aggressive about msrv.